scripts/commands/setrank.lua

```diff
@@ -0,0 +1,37 @@
+---------------------------------------------------------------------------------------------------
+-- func: setrank
+-- desc: Sets the players rank.
+---------------------------------------------------------------------------------------------------
+
+cmdprops =
+{
+    permission = 1,
+    parameters = "si"
+};
+
+function error(player, msg)
+    player:PrintToPlayer(msg);
+    player:PrintToPlayer("!setrank <player> <amount>");
+end;
+
+function onTrigger(player, target, rank)
+    local targ
+
+    if target == nil then
+        error(player, "Set who's rank?")
+        return
+    end
+
+    targ = GetPlayerByName(target)
+
+    if targ == nil then
+        error(player, string.format("Cannot find player: %s.", target))
+        return
+    end
+
+    if rank == nil or ( rank < 1 or rank > 10 ) then
+        error(player, "Improper rank passed.\nValid Values: 1 to 10")
```

Comment:
why not just 1 line? save muh precious vertical space. the game has wrapping built in for long lines / narrow user defines widths on menu.

Reply:
Just thought it was cleaner, but if you want it changed I will.

Reply:
I think
```lua
error(player, "Improper rank passed. Valid values: 1 to 10")
```
Looks just as well.
If our binding wasn't hard coded to "whatever nation you presently are on" I'd say make which rank your adjusting an option too.

Reply:
Done